scene_setup/Block block
--- 
float64 duration
bool success